app/assets/javascripts/partials/template_new.html

```diff
@@ -56,10 +56,10 @@ <h3 class="text-center no-print">New Template</h3>
     </ul>
     <ul class="nav nav-pills nav-stacked no-print">
       <li>
-        <input class="form-control" type="text" placeholder="New entry" ng-model="newContent" on-enter="addToEntries()" />
+        <input class="form-control" type="text" placeholder="New entry" ng-model="newContent" ng-blur="addToEntries()" on-enter="addToEntries()" />
       </li>
     </ul>
 
-    <button class="btn btn-primary col-md-4 col-md-offset-4 no-print" style="margin-top: 2em;" ng-click="save()">Create</button>
+    <button class="btn btn-primary col-md-4 col-md-offset-4 no-print" style="margin-top: 2em;" ng-mousedown="save()">Create</button>
```

Comment:
Should probably add a comment as to why mousedown is being used so no one removes it in the future for being odd.